vision/segmentation/plane_segmentation.go

```diff
@@ -188,27 +251,48 @@ func SegmentPlane(ctx context.Context, cloud pc.PointCloud, nIterations int, thr
 // PlaneSegmentation is an interface used to find geometric planes in a 3D space.
 type PlaneSegmentation interface {
 	FindPlanes(ctx context.Context) ([]pc.Plane, pc.PointCloud, error)
+	FindGroundPlanes(ctx context.Context) ([]pc.Plane, pc.PointCloud, error)
```

Comment:
There should only be 1 ground plane, right? So just call it `FindGroundPlane` and have it return just a pc.Plane rather than []pc.Plane. Like, I could see that there are 4 planes in a scene, two walls, the ground, and table. The ground and the table are both "ground planes" because they match the flat normal of the input vector. But the table should be an obstacle, and shouldn't be counted as a ground plane

Reply:
Makes sense, I think the idea was if your maxDistFromPlane is small, you would still want to catch stragglers/discrepancies, but hopefully they would be considered noise and filtered later anways.